packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -626,10 +626,17 @@ export function createProjectionNode<I>({
             this.nodes!.forEach(notifyLayoutUpdate)
             this.clearAllSnapshots()
 
-            // Flush any scheduled updates
+            /**
+             * Manually flush any pending updates. Ideally
+             * we could leave this to the following requestAnimationFrame but this seems
+             * to leave a flash of incorrectly styled content.
+             */
+            frameData.timestamp = performance.now()
```

Comment:
Do we also need to update frameData.delta here?

Reply:
Good point, I'll take a look